resources/views/auth/application/questions.blade.php

```diff
@@ -103,20 +103,27 @@
                         </div>
                     </div>
                     <x-input.textarea id="question_2" text="Miért kíván a Collegium tagja lenni?"
-                                      helper="≈300-500 karakter" :value="$user->application->question_2"/>
+                                      helper="≈300-500 karakter" maxlength="1000"
+                                      :value="$user->application->question_2"/>
                     <x-input.textarea id="question_3"
-                                      text="Tervez-e tovább tanulni a diplomája megszerzése után? Milyen tervei vannak az egyetem után?"
+                                      text="Tervez-e továbbtanulni a diplomája megszerzése után? Milyen tervei vannak az egyetem után?"
+                                      maxlength="1000"
```

Comment:
have you tested this in the ui?

Reply:
Yes, I did:) the only problem was that it did not count newlines the same way as the validation rule did.